3-extra/2-sorting-algorithm.js

```diff
@@ -13,9 +13,33 @@ Create a function called sortAges which:
 You don't have to worry about making this algorithm work fast! The idea is to get you to
 "think" like a computer and practice your knowledge of basic JavaScript.
 */
+function quicksort(array) {
+  if (array.length <= 1) {
+    return array;
+  }
 
-function sortAges(arr) {}
+  let pivot = array[0];
 
+  let left = [];
+  let right = [];
+
+  for (let i = 1; i < array.length; i++) {
+    array[i] < pivot ? left.push(array[i]) : right.push(array[i]);
+  }
+
+  return quicksort(left).concat(pivot, quicksort(right));
+}
+
+function sortAges(arr) {
```

Comment:
really well done on making it onto the extra folder and having passing tests! Just a quick note on .splice, remember it mutates the original array - so always better to make a copy of it first